examples/resources/hcp_hvn_route/resource.tf

```diff
@@ -0,0 +1,59 @@
+provider "aws" {
```

Comment:
The example here is for TGW attachment, I was thinking perhaps we could also include an example for creating an hvn route for a peering in this same file, but not sure if that would be sensible, wanted to hear what the team thinks

Reply:
Yeah I'm fine with either both examples or just peering, since I think that's slightly more commonly used.

Reply:
Added peering to example!